### PR TITLE
feat: add Azerbaijani translation

### PR DESCRIPTION
**Pull request by orkhan-huseyn**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

Added Azerbaijani translation into locale provider

### 💡 Background and solution

Added Azerbaijani translation into locale provider

### 📝 Changelog

Added Azerbaijani translation into locale provider

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/locale-provider/az_AZ.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+export default {
```
Comment (yoyo837) [inline components/locale-provider/az_AZ.tsx]:
```js
import locale from '../locale/az_AZ';

export default locale;
```

**Review by yoyo837 — CHANGES_REQUESTED** [review]
components/locale-provider/az_AZ.tsx: 
```js
import locale from '../locale/az_AZ';

export default locale;
```

File: components/locale/az_AZ.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+export default {
```
Comment (yoyo837) [inline components/locale/az_AZ.tsx]: image

Please refer to this code.

**Comment by orkhan-huseyn** [comment]
Hi, 
I have added `az_AZ` locale to `rc-picker` library and it has been merged.
Why am I getting this error on Circle CI?

```
components/date-picker/locale/az_AZ.tsx:1:28 - error TS2307: Cannot find module 'rc-picker/lib/locale/az_AZ'.

1 import CalendarLocale from 'rc-picker/lib/locale/az_AZ';
```

**Comment by yoyo837** [comment]
rc-picker has not released a new version

File: components/calendar/locale/az_AZ.tsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import tr_TR from '../../date-picker/locale/az_AZ';
```
Comment (yoyo837) [inline components/calendar/locale/az_AZ.tsx]: typo

Comment (orkhan-huseyn) [reply components/calendar/locale/az_AZ.tsx]: Fixed

**Comment by yoyo837** [comment]
`rc-pagination` need a PR for this.

And commits should be clean.

**Comment by orkhan-huseyn** [comment]
I have added the same locales to `rc-pagination` as well. And it has been [merged](https://github.com/react-component/pagination/pull/249) into master.

**Comment by yoyo837** [comment]
Also not released yet

**Comment by orkhan-huseyn** [comment]
Hi, when do we know that the depended packages are released new version?
Also, what should I do about commits? I already have 8 commits for a simple feature.

**Comment by yoyo837** [comment]
Your commits is mixed with other people's changes.

About dependent new version, you can watch its release.

image

**Comment by orkhan-huseyn** [comment]
That's because I have fethced and rebased before pushing. What should I do about them?

**Comment by yoyo837** [comment]
You can try rebase again.

**Comment by zombieJ** [comment]
`+ rc-picker@0.0.1-rc.6`

File: components/form/style/vertical.less [hunk 4]
```diff
@@ -24,6 +24,13 @@
   .@{form-prefix-cls}-item .@{form-prefix-cls}-item-label {
     .make-vertical-layout-label();
   }
+  .@{form-prefix-cls}-item {
+    .@{form-prefix-cls}-item-label,
+    .@{form-prefix-cls}-item-control {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+  }
```
Comment (zombieJ) [inline components/form/style/vertical.less]: Should not affect other files

**Comment by yoyo837** [comment]
I think you may need to send a new PR, please remember to pull the latest before that.

**Comment by orkhan-huseyn** [comment]
Alright. I am closing this PR then

**Pull request closed without merging**